webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, OnInit } from '@angular/core';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { select, Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { BaseListingComponent } from 'src/app/shared/base-listing.component';
+import { ReceiverSyncArchive } from './receiver-sync-archive';
+import { ReceiverSyncArchiveService } from './receiver-sync-archive.service';
+import { SyncArchiveLoaded } from './state/sync-archive.actions';
+import { GET_SYNC_ARCHIVE } from './state/sync-archive.reducer';
+
+@Component({
+  selector: 'app-archive',
+  templateUrl: './archive.component.html',
+  styleUrls: ['./archive.component.scss']
+})
+export class ArchiveComponent extends BaseListingComponent implements OnInit {
+
```

Comment:
Please always format your code to ensure you have no extra lines like here, the IDE has this feature built-in to format code

Reply:
?

Reply:
And this comment?